src/test/java/hu/akarnokd/rxjava/interop/RxJavaInteropV2SchedulerToV1SchedulerTest.java

```diff
@@ -0,0 +1,173 @@
+package hu.akarnokd.rxjava.interop;
+
+import io.reactivex.Scheduler;
+import io.reactivex.schedulers.TestScheduler;
+import org.junit.Test;
+import rx.functions.Action0;
+
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import static org.mockito.Mockito.*;
+
+public class RxJavaInteropV2SchedulerToV1SchedulerTest {
```

Comment:
Decided to make separate test class, `RxJavaInteropTest` becomes kinda hard to maintain, hope you don't mind that :)